packages/core/src/browser/test/mock-opener-service.ts

```diff
@@ -15,13 +15,17 @@
  ********************************************************************************/
 
 import { injectable } from 'inversify';
+import { Disposable } from './../../common/disposable';
 import { OpenerService, OpenHandler } from '../opener-service';
 
 /**
  * Mock opener service implementation for testing. Never provides handlers, but always rejects :)
  */
 @injectable()
 export class MockOpenerService implements OpenerService {
+    addHandler(openHandler: OpenHandler): Disposable {
+        throw new Error('MockOpenerService is for testing only.');
+    }
```

Comment:
minor: it can be removed since it is not longer mandatory.

Reply:
As it is optional I left it as is but will remove it now.